geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneQueriesIntegrationTest.java

```diff
@@ -290,6 +295,24 @@ public void queryJsonObject() throws Exception {
   }
 
   @Test()
+  public void testWaitUntilFlushedForException() throws Exception {
```

Comment:
Perhaps rename this be a bit more descriptive so a test maintainer can know what the test was expected to do when pass/failing.

Something like: waitUntilFlushThrowsIllegalStateExceptionWhenAEQNotFound